rewrite-weblogic/src/main/resources/META-INF/rewrite/weblogic-15.1.1.yaml

```diff
@@ -276,3 +277,21 @@ tags:
   - schemas
 recipeList:
   - com.oracle.weblogic.rewrite.WebLogicResourceDeploymentPlanXmlNamespace1412
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: com.oracle.weblogic.rewrite.ChangeJakartaInjectAPIDependencyScope
```

Comment:
I think that in this case, do not need to use the .jakarta. on the name, as is specific for weblogic